tortoise/fields.py

```diff
@@ -517,6 +528,20 @@ def __init__(
         self._generated = False
 
 
+def ManyToManyField(
+    model_name: str,
+    through: Optional[str] = None,
+    forward_key: Optional[str] = None,
+    backward_key: str = "",
+    related_name: str = "",
+    field_type: "Type[Model]" = None,  # type: ignore
```

Comment:
Technically the `field_type` is an internal API, and only ever set either later or during `__init__` (If I remember correctly). So we can now due to the separation of interface split up the public/private API quite well.

Reply:
Just removed field_type and ran the tests. All tests were successful!